examples/examples_nodejs_test.go

```diff
@@ -120,6 +120,16 @@ func TestAccCloudWatch(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccCloudWatchOidcManual(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:           filepath.Join(getCwd(t), "cloudwatchOidcManual"),
+			RunUpdateTest: true,
```

Comment:
Do you need RunUpdateTest? Does it work for you? Last time I spelunked there in this ProgramTest functionality it didn't really do anything useful.

Reply:
I just copied the test above and changed the target directory. I'd leave it alone since whether it's useful here or not, it's unrelated to this PR.